ruby-sdk/ext/eppo_client/src/client.rs

```diff
@@ -107,22 +113,21 @@ impl Client {
                     jitter: config.poll_jitter,
                 },
             );
-            Some((thread, poller))
+            Some(poller)
         } else {
             None
         };
 
-        let evaluator = Evaluator::new(EvaluatorConfig {
-            configuration_store: configuration_store.clone(),
-            sdk_metadata: SDK_METADATA,
-        });
+        let event_ingestion = config
+            .event_ingestion_config
+            .map(|config| config.spawn(background_thread.runtime()));
```

Comment:
in what scenario will `event_ingestion` be `None`? if SDKKey fails to parse the ingestion config?

Reply:
Correct
https://github.com/Eppo-exp/eppo-multiplatform/blob/c2c54f25f4754fa4b83b5896dcce945febd57dd2/eppo_core/src/event_ingestion/event_ingestion.rs#L35-L38